go-backend/db/recognition.go

```diff
@@ -0,0 +1,151 @@
+package db
+
+import (
+	"context"
+
+	logger "github.com/sirupsen/logrus"
+)
+
+const (
+	createRecognitionQuery = `
+		INSERT INTO recognitions (
+			core_value_id,
+			text,
+			given_for,
+			given_by,
+			given_at
+		)
+		VALUES ($1, $2, $3, $4, $5) returning id
+	`
+	showRecognitionQuery = "SELECT * FROM recognitions WHERE id=$1"
+	listRecognitionQuery = "SELECT * FROM recognitions ORDER BY given_at ASC"
+)
+
+func listRecognitionsFilterByOneFilterQuery(column_name string) string {
```

Comment:
this is not a good way to create dynamic queries, instead where parameters can be added dynamically on the basis of number of filters required through a loop